backports/py3-rosdistro/force-to-enable-alpine-platform.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/src/rosdistro/distribution_file.py b/src/rosdistro/distribution_file.py
+index c330329..e6265dd 100644
+--- a/src/rosdistro/distribution_file.py
++++ b/src/rosdistro/distribution_file.py
+@@ -74,6 +74,15 @@ class DistributionFile(object):
+                     assert os_code_name not in self.release_platforms[os_name], "Distribution '%s' specifies the os_code_name '%s' multiple times for the os_name '%s'" % (self.name, os_code_name, os_name)
+                     self.release_platforms[os_name].append(os_code_name)
+ 
++        # Tweak for alpine platform
++        self.release_platforms['alpine'] = []
++        if name == 'kinetic':
++            self.release_platforms['alpine'].append('3.7')
```

Comment:
`kinetic` can probably be removed.

Reply:
I think name resolution is better to be remained.